openid-4-verifiable-presentations-1_0.md

```diff
@@ -823,6 +823,7 @@ This specification defines how the Verifier can determine Credential formats, pr
 This specification defines new metadata parameters according to [@!RFC8414].
 
 * `presentation_definition_uri_supported`: OPTIONAL. Boolean value specifying whether the Wallet supports the transfer of `presentation_definition` by reference, with true indicating support. If omitted, the default value is true.
+* `conditional_credential_request_supported`: OPTIONAL. Boolean value specifying whether the Wallet supports the conditional credential request flow when combined with SIOPv2. If omitted, the default value is `false`.
```

Comment:
why is this parameter needed? is it pretty much wallet indicating support for SIOPv2? if so, maybe make that clearer? tho not sure `siopv2_supported` makes sense..

Reply:
it indicates that the wallet may need extra capabilities to handle this optionality
I am open to removing it if you feel it is in unnecessary